wgsl/index.bs

```diff
@@ -10022,6 +10040,8 @@ Note: Address space in WGSL is equivalent to storage class in SPIR-V.
 model memory semantics|memory semantics=] and `WorkgroupMemory` semantics.
 [[#sync-builtin-functions|storageBarrier]] uses `AcquireRelease` [=memory model
 memory semantics|memory semantics=] and `UniformMemory` semantics.
+[[#sync-builtin-functions|textureBarrier]] uses `AcquireRelease` [=memory model
+memory semantics|memory semantics=] and `ImageMemory` semantics.
 
 Note: A combined `workgroupBarrier` and `storageBarrier` uses `AcquireRelease`
```

Comment:
nit: In principle this should also talk about combining textureBarrier with the others.
But this is a note, so it's non-normative, and secondly the word "combine" is somewhat ambiguous.

Reply:
I thought about that, but I didn't want to cover all the possibilities. I considered trying to make it generic, but it didn't really seem worth it.